opsdroid/message.py

```diff
@@ -22,9 +22,11 @@ class Message(NewMessage):
     # Skip registering this class with Event
     _no_register = True
 
-    def __init__(self, text, user, room, connector, raw_message=None):  # noqa: D401
+    def __init__(
```

Comment:
we don't want to change this file as it's only provided for backwards compatibility.